### PR TITLE
Drag and drop feedback

### PR DESCRIPTION
**Pull request by bart-de-jonge**
Added feedback pane while drag-and-dropping
Perfected smart-snapping

### CODE REVIEW

File: src/main/java/gui/RootCenterArea.java [hunk 1]
```diff
@@ -107,6 +112,7 @@ public SnappingPane getMyPane(double x, double y) {
             Bounds bounds = pane.localToScene(pane.getBoundsInLocal());
             if (bounds.contains(x, y)) {
                 if (((y - bounds.getMinY()) * 2) > pane.getHeight()) {
+                    System.out.println("DEBUG SHI T = " + ((y - bounds.getMinY()) * 2));
```
Comment (geenen124) [inline src/main/java/gui/RootCenterArea.java]: Maybe remove this print statement?

File: src/main/java/gui/RootCenterArea.java [hunk 2]
```diff
@@ -90,6 +90,11 @@ private void addPanes() {
         for (int i = 0; i < numberOfCounts; i++) {
             for (int j = 0; j < numberOfTimelines; j++) {
                 SnappingPane pane = new SnappingPane(i, j, 200, 50);
+//                pane.setVisible(true);
+//                pane.setStyle("-fx-border-style: solid inside;"
```
Comment (mennoo1996) [inline src/main/java/gui/RootCenterArea.java]: Is this code needed later? If not, it's better to remove it

File: src/main/java/gui/TimetableBlock.java [hunk 3]
```diff
@@ -170,16 +164,72 @@ private void onMouseDraggedHelper(MouseEvent event) {
...
+            feedbackPane.setVisible(false);
+        }
+
         // store current mouse position for next mouse movement calculation
         mouseCurrentXPosition = event.getSceneX();
         mouseCurrentYPosition = event.getSceneY();
     }
 
     /**
+     * Snap tha targetregion to a grid using the model provided by the mappingPane.
```
Comment (martijnjanssen) [inline src/main/java/gui/TimetableBlock.java]: Typo `the`.

**Comment by mennoo1996** [comment]
Good working fixing all the issues!

**Pull request merged**